src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -75,7 +80,11 @@
                   "maximumWarning": "6kb"
                 }
               ],
-              "optimization": true,
+              "optimization": {
+                "styles": {
+                    "inlineCritical": false
```

Comment:
What's the reason for this change? is there any related issue that can be reproducible?

Reply:
there was. the one I mentioned in the description. https://github.com/angular/angular-cli/issues/20760 Although now its seems to be resolved (atleast closed). I'll test it again.

Reply:
@nizamial09 Thanks. i tested the bundle generation and it was successful:
```
✔ Browser application bundle generation complete.
✔ Localized bundle generation complete.
✔ Copying assets complete.
✔ Index html generation complete.
```

Reply:
Perfect! Then I'll take care of it too. Thanks @alfonsomthd

Reply:
@alfonsomthd today I made the change and repushed. I noticed that this is failing on dashboard-cephadm job. So my guess is in dev environment it this error is not happening since optimization is false but on prod environment this is happening.
```
✔ Browser application bundle generation complete.
✔ Browser application bundle generation complete.
- Generating localized bundles...
✔ Localized bundle generation complete.
- Copying assets...
✔ Copying assets complete.
- Generating index html...
✖ Index html generation failed.
undefined:2:146736: property missing ':'
```

Reply:
https://jenkins.ceph.com/job/ceph-dashboard-cephadm-e2e/1338/console

Reply:
@nizamial09 After investigating the issue I found that [upgrading to ngx-toastr v14.1.2 fixes the issue](https://github.com/scttcper/ngx-toastr/pull/901):
```
Option "--prod" is deprecated: Use "--configuration production" instead.
✔ Browser application bundle generation complete.
✔ Localized bundle generation complete.
✔ Copying assets complete.
⠋ Generating index html...1 rules skipped due to selector errors:
  .custom-file-input:lang(en)~.custom-file-label -> unmatched pseudo-class :lang
✔ Index html generation complete.
```
Can you confirm that the upgrade does not have any impact?

Reply:
Oh okay. I'll test it out. Thanks @alfonsomthd

Reply:
just noticed the error `.custom-file-input:lang(en)~.custom-file-label -> unmatched pseudo-class :lang`

Reply:
This warning is related to the bootstrap version 4 (it foes not happen with v5),
but upgrading bootstrap is out of scope of this PR.

Reply:
@alfonsomthd I've upgraded the ngx-toaster and also updated the ng-bootstrap to 10 and noticed this error is not there. I've also updated the minor version of angular using `npx-update` command
```
✔ Browser application bundle generation complete.
✔ Browser application bundle generation complete.
- Generating localized bundles...
✔ Localized bundle generation complete.
- Copying assets...
✔ Copying assets complete.
- Generating index html...
✔ Index html generation complete.
```